.github/workflows/push.yml

```diff
@@ -89,6 +99,10 @@ jobs:
         uses: actions/checkout@v3
       - name: apt
         uses: ./.github/actions/apt-x32
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1.2
+        with:
+          key: "${{github.job}}-${{hashFiles('main/php_version.h')}}"
```

Comment:
Is this intentionally different? / intentionally using github.job?

Reply:
Yes. In these two jobs, the job id and the job name are identical. Not so with the first job - there's an override for the name.

Reply:
Technically you could still use `${{github.job}}` instead of duplicating `LINUX_X64` above, right? Not a big deal, just for consistency.

Reply:
Yeah you're right, didn't even notice that. It's fine then, we can adjust it everywhere if desired.